cdap-docs/included-applications/source/etl/creating.rst

```diff
@@ -115,20 +115,20 @@ Stream after performing a projection transformation::
   }
 
 
-The ETL Real-Time Template expects an instance property that will create *N* instances of the
-adapter that run concurrently. In Standalone CDAP mode, this is implemented as multiple threads;
-while in Distributed CDAP mode, it will create different YARN containers. The
-number of instances of a real-time adapter cannot be changed during runtime.
+An ETL Real-Time Application expects an instance property that will create *N* instances
+of the worker that run concurrently. In Standalone CDAP mode, this is implemented as
+multiple threads; in Distributed CDAP mode, it will create different YARN containers. The
+number of worker instances of a real-time application cannot be changed during runtime.
```

Comment:
If the number of instances needs to be changed, the worker must be stopped, then the application configuration must be updated to the new number of instances.